uwblueprint/jump-math
# Omar/delete test by id

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Delete-test-0cce32099ffb4bc99409082ebd983a00)



## Implementation description
* created interface for deleteTest
* implemented deleteTest
* added two tests to test deleteTest behaviour when test id is found and is missing.



## Steps to test
1.run yarn test and make sure both tests, "deleteTest" and "deleteTest" when id not found", pass



## What should reviewers focus on?
* 


## Checklist
